src/plugins/intel_cpu/src/nodes/executors/deconv.hpp

```diff
@@ -22,6 +22,9 @@ struct DeconvAttrs {
     std::vector<ptrdiff_t> paddingR;
     ov::CoordinateDiff outputPadding;
     bool withBiasesParam = false;
+#if defined(OV_CPU_WITH_ACL)
+    bool aclFastMath = false;
```

Comment:
I would prefer to avoid backend specific params in common attributes structure. 
Doesn't ACL Deconv Executor have direct access to GraphConfig?

Reply:
It doesn't.

Reply:
Ok, I think we can merge this as temp solution, but in general it should be refactored. 
I would expect to have some backend related properties inside ExecutorContext. @EgorDuplensky what is your opinion?